tools/RAiDER/llreader.py

```diff
@@ -32,6 +32,21 @@ def bounds(self):
     def projection(self):
         return self._proj
     
+    def add_buffer(self, buffer):
```

Comment:
One thing which is a pain, maybe for future is that adding a buffer might take polygons over the dateline even if they were not in the first place. In general, we need logic to handle the dateline